app/assets/javascripts/miq_application.js

```diff
@@ -734,6 +734,23 @@ function miqChartMenuClick(itemId) {
   }
 }
 
+function miqRESTAjaxButton(url, button) {
+  var form = $(button).parents('form:first')[0];
+  if (form) {
+    $(form).submit(function(e) {
+      e.preventDefault();
+      return false;
+    });
+    miqJqueryRequest(form.action, {
+      beforeSend: true,
+      complete: true,
+      data: $(form).serialize()
```

Comment:
@tenderlove I noticed that ```$(form).serialize()``` seems to be working differently for ```create``` and ```update```.

```update``` I think, is working correctly since it is serializing the entire form.
```
[----] I, [2015-08-27T12:19:05.820503 #54965:3fcdc0860204]  INFO -- : Started PATCH "/ems_cloud/206000000000083" for 127.0.0.1 at 2015-08-27 12:19:05 -0700
[----] I, [2015-08-27T12:19:05.854982 #54965:3fcdc0860204]  INFO -- : Processing by EmsCloudController#update as JS
[----] I, [2015-08-27T12:19:05.855167 #54965:3fcdc0860204]  INFO -- :   Parameters: {"utf8"=>"✓", "authenticity_token"=>"5RpeLKkq/CWbQGWqFu4nEhT4fHk1P4cJn2qG+js9fqUQYDvgfuLrDxU+QI7RNHc50FhbBNv8zabFWtELD20NyA==", "button"=>"save", "name"=>"bbn2", "provider_region"=>"eu-central-1", "port"=>"", "default_userid"=>"77y", "default_password"=>"[FILTERED]", "default_verify"=>"[FILTERED]", "metrics_userid"=>"", "metrics_password"=>"[FILTERED]", "metrics_verify"=>"[FILTERED]", "amqp_userid"=>"", "amqp_password"=>"[FILTERED]", "amqp_verify"=>"[FILTERED]", "ssh_keypair_userid"=>"", "ssh_keypair_password"=>"[FILTERED]", "id"=>"206000000000083"}
```
But ```create``` on the other hand, seems to be excluding the credential fields 
```
[----] I, [2015-08-27T12:17:08.639603 #54965:3fcdc0860204]  INFO -- : Started POST "/ems_cloud" for 127.0.0.1 at 2015-08-27 12:17:08 -0700
[----] I, [2015-08-27T12:17:08.674333 #54965:3fcdc0860204]  INFO -- : Processing by EmsCloudController#create as JS
[----] I, [2015-08-27T12:17:08.674510 #54965:3fcdc0860204]  INFO -- :   Parameters: {"utf8"=>"✓", "authenticity_token"=>"ZiTxXkmPCugC1Vnd2wTLW+TAzAPUcadxofqfCflcqOyTXpSSnkcdwoyrfPkc3ptwIGDrfjqy7d77ysj4zQzbgQ==", "button"=>"add", "name"=>"9900", "server_emstype"=>"ec2", "provider_region"=>"ap-southeast-2", "port"=>""}
```
And since its excluding credentials from being serialized, there is no ```Authentication``` record created for ```create```.